.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+---
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
```

Comment:
To simplify things, I've added a dependabot config.
This doesn't bump JS dependencies as Dependabot doesn't support Bun yet, but it will bump actions.
This is set to weekly updates, which felt like a good balance between the spam of daily (me: 😞) and the slow pace of monthly (as you're currently getting potentially-insecure instant updates).